lapce-data/src/data.rs

```diff
@@ -949,7 +949,7 @@ impl LapceTabData {
             LapceWorkbenchCommand::OpenFolder => {
                 if !self.workspace.kind.is_remote() {
                     let event_sink = ctx.get_external_handle();
-                    let tab_id = self.id;
```

Comment:
This was not actually used? As far as I can tell.

Reply:
#439 removes the `allow(unused)` that hid this.